jdi-light-material-ui/src/main/java/com/epam/jdi/light/material/elements/inputs/Checkbox.java

```diff
@@ -3,16 +3,26 @@
 import com.epam.jdi.light.asserts.generic.HasAssert;
 import com.epam.jdi.light.common.JDIAction;
 import com.epam.jdi.light.elements.base.UIBaseElement;
+import com.epam.jdi.light.elements.common.UIElement;
 import com.epam.jdi.light.elements.interfaces.base.HasCheck;
 import com.epam.jdi.light.elements.interfaces.base.HasClick;
 import com.epam.jdi.light.material.asserts.inputs.CheckboxAssert;
 
 public class Checkbox extends UIBaseElement<CheckboxAssert> implements HasClick, HasCheck,
-    HasAssert<CheckboxAssert> {
+    HasAssert<CheckboxAssert>{
+
+    public UIElement getCheckBoxes() {
+        return this.core();
+    }
```

Comment:
Is it possible to delete this method and just to use "core()._method_" like it was before?

Reply:
It's possible to do this, I have done it now